test/test_json_config.py

```diff
@@ -128,6 +128,121 @@ class TestJsonConfig(unittest.TestCase):
       }
     }'''
 
+    windows_json_file = '''{
+      "config": {
+        "hostname": "mgarewal",
+        "endpoint": "data.logentries.com",
+        "user-key": "3d7946d4-0d97-11e7-9b83-6c0b84a93740",
+        "agent-key": "498fa2ba-0d97-11e7-9b83-6c0b84a93740",
+        "api-key": "555e7094-0d97-11e7-9b83-6c0b84a93740",
+        "metrics": {
+          "system-stat-token": "627d011e-0d97-11e7-9b83-6c0b84a93740",
+          "system-stat-enabled": "true",
+          "metrics-interval": "60s",
+          "metrics-cpu": "system",
+          "metrics-vcpu": "core",
+          "metrics-mem": "system",
+          "metrics-swap": "system",
+          "metrics-net": "sum eth0",
+          "metrics-disk": "sum sda4 sda5",
+          "metrics-space": "/"
+        },
+        "logs": [
+          {
+            "name": "GreenLog",
+            "token": "a7f9625e-0d98-11e7-9b83-6c0b84a93740",
+            "path": "/var/log/GreenLog",
+            "enabled": "true"
+          }
+        ],
+        "windows-eventlog":{
+        "enabled": true,
+        "token": "e684dc57-7240-4669-aa67-317e5493040a"
+       }
+      }
+    }'''
+
+    windows_json_file_incorrect_token = '''{
+      "config": {
+        "hostname": "mgarewal",
+        "endpoint": "data.logentries.com",
+        "user-key": "3d7946d4-0d97-11e7-9b83-6c0b84a93740",
+        "agent-key": "498fa2ba-0d97-11e7-9b83-6c0b84a93740",
+        "api-key": "555e7094-0d97-11e7-9b83-6c0b84a93740",
+        "metrics": {
+          "system-stat-token": "627d011e-0d97-11e7-9b83-6c0b84a93740",
+          "system-stat-enabled": "true",
+          "metrics-interval": "60s",
+          "metrics-cpu": "system",
+          "metrics-vcpu": "core",
+          "metrics-mem": "system",
+          "metrics-swap": "system",
+          "metrics-net": "sum eth0",
+          "metrics-disk": "sum sda4 sda5",
+          "metrics-space": "/"
+        },
+        "logs": [
+          {
+            "name": "GreenLog",
+            "token": "a7f9625e-0d98-11e7-9b83-6c0b84a93740",
+            "path": "/var/log/GreenLog",
+            "enabled": "true"
+          }
+        ],
+        "windows-eventlog":{
+        "enabled": true,
+        "token": "incorrect_token"
+       }
+      }
+    }'''
```

Comment:
The original json configuration is set up so that original logentries log following feature is enabled by default. 
This work is an additional feature for windows-event log following, which I have set as optional.
If we want the user to be able to specify between the two, lets set up a ticket for that and discuss in spring planning.